libs/design-system-core/src/components/ErrorLoader/index.tsx

```diff
@@ -36,7 +39,16 @@ export type ErrorLoaderProps = React.PropsWithChildren<{
  * ```
  **/
 const ErrorLoader: React.FC<ErrorLoaderProps> = ({ children, ...props }) => {
-  const { type, publicImgPath = '/images' } = props;
+  const {
+    type,
+    publicImgPath = '/images',
```

Comment:
We can use the env variable for this that way we can set and change it from one place. I will create a ticket because this pattern is used in many places throughout the app and we can tackle them all at once.

Reply:
I would say that this configuration could come from context, having it from env is too rigid since every consumer might define a different path. Setting a default value for this is ok

Reply:
I agree. We can use theme context. I have created a ticket for this on the backlog.